inc/blocks/namespace.php

```diff
@@ -157,6 +159,86 @@ function on_save_post( int $post_ID, WP_Post $post, bool $update ) : void {
 	}
 }
 
+/**
+ * Synchronise any XBs on the widget page with a shadow post type.
+ *
+ * @param instance $instance Widget Object.
+ * @param new_instance $new_instance Widget Object.
+ * @param old_instance $old_instance Widget Object
+ * @param widget $widget Current Widget Instance
```

Comment:
This isn't proper docblock syntax, not sure if it was generated but you should know what the components are and how they should be written, see this guide for an overview: https://www.open-emr.org/wiki/index.php/How_to_Document_Your_Code_Properly

For the `@param` annotation you'll see in other examples in the file that it's `@param <type> <variable name> <description>`.

- The `type` is any valid PHP variable type such as `array`, `bool`, `string` or a class name etc...
- The `description` should end in a full stop and be a bit more descriptive whenever possible

The same goes for the return type "object", although valid it is not what is returned by this filter, it's actually an `array`. See the filter docs: https://developer.wordpress.org/reference/hooks/widget_update_callback/